pkg/crc/api/handlers.go

```diff
@@ -29,9 +29,12 @@ type Telemetry interface {
 	UploadAction(action, source, status string) error
 }
 
+type loggerResult struct {
```

Comment:
This is part of `types.go` so why don't we update it there instead of creating here again?

Reply:
All the types in the `types.go` were already moved to `client/types.go` since most were relevant for the client and the responses from the handlers, this `loggerResult` was the only type left there and is only used in the handlers. its just a refactor to bring it here where its used, but i can keep the `types.go` file and update the struct there if you prefer that way